app/controllers/admin/articles_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Admin
+  class ArticlesController < ApplicationController
+  	before_filter :admin_authorize!
+  	def index
+  		@article = Article.all
+    end		
+  	def new
+  		@article = Article.new
+  	end	
+    
+  	def destroy
+  		
+			 @article = Article.find(params[:id])
+			 
+				@article = Article.find(params[:id]).destroy
+				
+				if Article.exists?(@article) == false
+					
+					redirect_to articles_path, :notice => "Article successfully deleted"
+				else
+					render :edit, :notice => "Article did not delete"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.